app/templates/_view_suppliers_agreements.html

```diff
@@ -1,86 +1,31 @@
-      {% endcall %}
-      {% call summary.field() %}
-        <a href="{{ url_for('.view_supplier_declaration', supplier_id=item.id, framework_slug='g-cloud-8') }}">Edit <span class="visually-hidden">G-Cloud 8 </span>declaration</a>
-      {% endcall %}
-      {% call summary.field() %}
-        <a href="{{ url_for('.view_supplier_declaration', supplier_id=item.id, framework_slug='digital-outcomes-and-specialists-2') }}">Edit <span class="visually-hidden">Digital Outcomes and Specialists 2 </span>declaration</a>
-      {% endcall %}
-      {% call summary.field() %}
-        <a href="{{ url_for('.view_supplier_declaration', supplier_id=item.id, framework_slug='g-cloud-9') }}">Edit <span class="visually-hidden">G-Cloud 9 </span>declaration</a>
-      {% endcall %}
-      {% call summary.field() %}
-        <a href="{{ url_for('.view_supplier_declaration', supplier_id=item.id, framework_slug='g-cloud-10') }}">Edit <span class="visually-hidden">G-Cloud 10 </span>declaration</a>
-      {% endcall %}
-    {% endcall %}
-
-    {% call summary.row(no_border=True) %}
-      {% call summary.field() %}
-        ⬇ <a href="{{ url_for('.download_agreement_file', supplier_id=item.id, framework_slug='g-cloud-7', document_name=agreement_filename) }}" ><span class="visually-hidden">Download G-Cloud 7 </span>Agreement</a>
-      {% endcall %}
-      {% call summary.field() %}
-        ⬇ <a href="{{ url_for('.download_agreement_file', supplier_id=item.id, framework_slug='digital-outcomes-and-specialists', document_name=agreement_filename) }}" ><span class="visually-hidden">Download Digital Outcomes and Specialists </span>Agreement</a>
-      {% endcall %}
-      {% call summary.field(rowspan=3) %}
-        <a href="{{ url_for('.view_signed_agreement', supplier_id=item.id, framework_slug='g-cloud-8') }}">View agreement <span class="visually-hidden"> for G-Cloud 8</span></a>
-      {% endcall %}
-      {% call summary.field(rowspan=3) %}
-        <a href="{{ url_for('.view_signed_agreement', supplier_id=item.id, framework_slug='digital-outcomes-and-specialists-2') }}">View agreement <span class="visually-hidden"> for Digital Outcomes and Specialists 2</span></a>
-      {% endcall %}
-      {% call summary.field(rowspan=3) %}
-        <a href="{{ url_for('.view_signed_agreement', supplier_id=item.id, framework_slug='g-cloud-9') }}">View agreement <span class="visually-hidden">for G-Cloud 9</span></a>
-      {% endcall %}
-      {% call summary.field(rowspan=3) %}
-        <a href="{{ url_for('.view_signed_agreement', supplier_id=item.id, framework_slug='g-cloud-10') }}">View agreement <span class="visually-hidden">for G-Cloud 10</span></a>
-      {% endcall %}
-    {% endcall %}
-
-    {% call summary.row(no_border=True) %}
-      {% call summary.field() %}
-        ⬇ <a href="{{ url_for('.download_signed_agreement_file', supplier_id=item.id, framework_slug='g-cloud-7') }}"><span class="visually-hidden">Download G-Cloud 7 </span>Signed agreement</a>
```

Comment:
I vaguely remember from the Admin & Automation mission that we wanted to get rid of these  'upload'/'download' arrows but didn't for some reason, and I can't remember what that was. It might have been to help users easily spot them given the poor layout of the page, which I guess now isn't a problem?

Reply:
Hmm...i would guess that it would that it would have made the text more cramped in the old layout? Hopefully (🤞🏻) those old frameworks can be removed if we here back from CCS. Either way, Kuba came up with the design and I guess he would have shouted if he thought them still necessary?

Reply:
Cool - if Kuba is happy then I am happy 😸